core/src/main/java/com/graphhopper/routing/util/NameSimilarityEdgeFilter.java

```diff
@@ -171,9 +171,17 @@ public final boolean accept(EdgeIteratorState iter) {
         return isJaroWinklerSimilar(pointHint, edgeName);
     }
 
-    private static BBox createBBox(NodeAccess na, EdgeIteratorState edgeState) {
-        return BBox.fromPoints(na.getLat(edgeState.getBaseNode()), na.getLon(edgeState.getBaseNode()),
-                na.getLat(edgeState.getAdjNode()), na.getLon(edgeState.getAdjNode()));
+    private static BBox createBBox(EdgeIteratorState edgeState) {
```

Comment:
This method was no longer used since #1950, but it did the same wrong thing (exclude pillar nodes)